docs/2_4_common_schema.adoc

```diff
@@ -1743,9 +1743,9 @@ The following <<dependenciesKind>> is only allowed for floating point latexmath:
 
 ==== Terminals and Icons [[fmiTerminalsAndIcons]]
 
-Terminals define semantic groups of variables to ease connecting compatible signals on the system level.
+Terminals define semantic groups of variables to ease connecting compatible variables on the system level.
```

Comment:
I'd either write (a) "Terminals define semantic groups of variables." or (b) "Terminals define semantic groups of variables. Such groups can for example be used to ease connecting compatible variables on the system level."
I prefer (a) since later on "...into logical groups, e.g.,  to ease connections on a system level..." is used anyway.